src/views/workflow-summary-tab/workflow-summary-tab-details/workflow-summary-tab-details.styles.ts

```diff
@@ -0,0 +1,37 @@
+import type {
+  StyletronCSSObject,
+  StyletronCSSObjectOf,
+} from '@/hooks/use-styletron-classes';
+
+const cssStylesObj = {
+  pageContainer: (theme) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.sizing.scale500,
+  }),
+  detailsRow: (theme) => ({
```

Comment:
Instead of defining custom styling for the table rows, it would be better to use https://baseweb.design/components/table-semantic/ and override TableHead to display: 'none'.

`overrides={{
        TableHead: {
          style: ({ $theme }) => ({
            display: "none"
          })
        }
      }}`

Reply:
I'm actually going to make a very similar component for Domain Metadata. If you'd like, you can go ahead with this implementation and I'll replace it when working on Metadata.

Reply:
It didn't seem to be a use case for a table and it provides better responsiveness (no scroll bars) and smaller bundle size (- 6kb)